### PR TITLE
Docker compose example.

### PR DESCRIPTION
**Pull request by yoanisgil**
Provide an example for docker-compose

### CODE REVIEW

**Comment by vishh** [comment]
ok to test

**Comment by yoanisgil** [comment]
Actually I think we can improve a bit more the compose file and link to
cadvisor container. This way the hosts file will point to a working host.
What do you think ?

File: deploy/docker-compose/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+heapster:
+    image: kubernetes/heapster:v0.14.3
+    ports:
+        - "8002:8002"
```
Comment (vishh) [inline deploy/docker-compose/docker-compose.yml]: This should be `8082` I think.

Comment (yoanisgil) [inline deploy/docker-compose/docker-compose.yml]: Yes. I will fix this alone with adding a cadvisor service in the compose
file

**Comment by vishh** [comment]
Just one comment. Otherwise LGTM.

**Comment by yoanisgil** [comment]
Ok. Please do not merge until I get cAdvisor service integrated and fix the port issue ;)

**Comment by vishh** [comment]
Ack. Just ping the PR once cAdvisor service is added!

**Comment by yoanisgil** [comment]
@vishh  there you are. We now have a fully functional working example with docker-compose. Users just need fire `docker-compose up` and they will be all set. I think we should link this somewhere in the docs so people know that this thing exists and can benefit from it.

**Comment by vishh** [comment]
Thanks @yoanisgil! Yes, we should place a link to the compose README from the main page.

**Comment by vishh** [comment]
Will merge once the e2e passes.

**Comment by yoanisgil** [comment]
@vishh Where can I check the output from e2e-gce? I would like to know why the tests are failing but when I click here:

http://104.154.52.74/job/heapster-e2e-gce/1453/

I get a 404. Any ideas?

**Comment by vmarmol** [comment]
The Jenkins build is currently broken sadly :( need to take a look. Feel free to ignore it for now. I don't think this PR would affect it anyways.

**Comment by vishh** [comment]
@cadvisorJenkinsBot: test this please

**Comment by vishh** [comment]
LGTM. Merging. Sorry for the delay @yoanisgil. I was gone for a few weeks.

**Pull request merged**